sonic-ledd/scripts/ledd

```diff
@@ -75,15 +79,20 @@ class DaemonLedd(DaemonBase):
             logger.log_error("Failed to load ledutil: %s" % (str(e)), True)
             sys.exit(LEDUTIL_LOAD_ERROR)
 
-        # Open a handle to the Application database
-        appl_db = daemon_base.db_connect("APPL_DB")
+        # Get the namespaces in the platform
+        namespaces = multi_asic.get_namespaces()
```

Comment:
if it's not a multi-asic platform, does this return only one namespace?

Reply:
yes, if it is not a multi-asic platform it returns back "" EMPTY_NAMESPACE which signifies linux host. 
Please take a look at the related PR https://github.com/Azure/sonic-buildimage/pull/4932